src/util.rs

```diff
@@ -251,6 +251,32 @@ pub fn access_of(register: &Register) -> Access {
         )
 }
 
+/// Turns `n` into an unsuffixed separated hex ident
+pub fn hex(n: u32) -> Ident {
```

Comment:
nit: `Ident` is for (parsed) identifiers. I see that there's no `Lit` variant for hexadecimal so we can't use that. I think returning `Tokens` would be the most correct choice after `Lit`. Could you change this to `Tokens`? (I think you would have to do something like `let mut ts = Tokens::new(); ts.append("0xdead_beef"); ts`

Reply:
I will try that.